### ISSUE TITLE
RStudio and quarto-cli >=0.9.130

### ISSUE THREAD

**Issue by trap000d** [quarto-dev/quarto-cli#458]
HI there,

I'm maintaining RStudio in Arch Linux AUR, so had to adopt quarto-cli as well. 
https://aur.archlinux.org/packages/quarto-cli

quarto-cli>=0.9.130 doesn't rendering projects in RStudio resulting the following error message:

```
ERROR: Cannot serve project of type 'default' (try using project type 'website').
```
I've tried to bisect it - v0.9.128 works OK

I'm aware that rstudio sources picking up quarto-cli 0.9.80, however nested package embedding just for compatibility looks for me sub-optimal. On the other hand, freezing package at some version in AUR doesn't looking good as well.

I would suggest either set `stable` tag on quarto-cli releases (see e.g. how it's done at laurent22/joplin) so I could maintain something like `quarto-cli-stable`

Regards.

**Comment by jjallaire** [comment]
I've got a repro of this, hopefully can fix very quickly.

Right now the `latest` Quarto release is the stable release (this will change relatively soon and we'll have a higher threshold around stable releases). The version in RStudio is just whatever point at which we decided to update the bundle (it doesn't represent stable).

In terms of managing the relationship in AUR, I would configure/build without the embedded Quarto and then make the AUR Quarto package a dependency (this _should_ work, if it doesn't then let us know and we'll adjust).

**Comment by jjallaire** [comment]
Fixed here: https://github.com/quarto-dev/quarto-cli/commit/ee1c0a9d705497145ba6111027c1e1f9361d0ffb (will be in the next binary release, action running right now)

**Comment by trap000d** [comment]
> I've got a repro of this, hopefully can fix very quickly.

I can confirm, it's fixed in 0.9.135

> 
> Right now the `latest` Quarto release is the stable release (this will change relatively soon and we'll have a higher threshold around stable releases). The version in RStudio is just whatever point at which we decided to update the bundle (it doesn't represent stable).

OK, got it. 

> 
> In terms of managing the relationship in AUR, I would configure/build without the embedded Quarto and then make the AUR Quarto package a dependency (this _should_ work, if it doesn't then let us know and we'll adjust).

Yes, this is exactly what I do in `rstudio-desktop` package: `quarto-cli` is optional dependency of `rstudio-desktop` - if it's installed, rstudio will be built with quarto support. Yet I couldn't make it a dependency due to outdated `pandoc` and `quarto-cli`
(though `pandoc-bin` and `quarto-cli-bin` are usually up to date, some people have strong concerns against binary packages). Might give a try to next RStudio release.

Thanks for quick fix :)
